pkg/server/server_test.go

```diff
@@ -787,3 +854,206 @@ func etcdoperator_v0_9_2(channel string, addSkipsReplaces, addExtraProperties bo
 	}
 	return b
 }
+
+func getCockroachBundle() *api.Bundle {
```

Comment:
If this is static data, why is it behind a function?

Reply:
It is mimicking an existing test pattern here, c.f. https://github.com/operator-framework/operator-registry/pull/1164/files#diff-fe7a244e18e92d8f347af17363209172e1d79224f086b0585a1bbeef3b626c4eR777

Reply:
Sorry, but I'm not sure that is sufficient justification - is there a reason that is independent of that?

Reply:
I'm anticipating that if there was a need to pivot the original, we might want to do that as well in the future.  I'm not doing it right now, but trying to match the idiom of the existing tests in a manner that lets them expand in a similar manner.